docs/user_guide/assessment.rst

```diff
@@ -258,6 +258,241 @@ across groups and also the difference and ratio between the maximum and minimum:
+capture the costs of missclassification disparities.
+
+.. _group_fairness:
+
+Considerations for conditional group fairness
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In some cases, we may observe a trend in several groups of data, but that 
+trend may disappear or reverse when groups are combined. Known as  
+`Simpson's Paradox <https://en.wikipedia.org/wiki/Simpson%27s_paradox>`_, this 
+outcome may appear when observing disparate outcomes across groups. A 
+famous example of Simpson's Paradox is a study of 1973 graduate school 
+admissions to the University of California, Berkley :footcite:`bickel1975biasinadmissions`. 
+The study showed that when observing admissions by gender, men applying were 
+more likely than women to be accepted. However, drilling down into admissions 
+by department revealed that women tended to apply to departments with more 
+competitive admissions requirements, whereas men tended to apply to less 
+competitive depoartments. The more granular analysis showed only four out of 
+85 departments exhibited bias against women, and six departments exhibited 
+bias towards men. In general, the data indicated departments exhibited a bias 
+in favor of minority-gendered applicants, which is opposite from the trend 
+observed in the aggregate data.
+
+This phenomenon is important to fairness evaluation because metrics like 
+demographic parity may be different when calculated at an aggregate level and  
+within more granular categories. In the case of demographic parity, we might 
+need to review 
+:math:`\E[h(X) \given A=a, D=d] = \E[h(X) \given D=d] \quad \forall a` 
+where D represents the feature(s) within X across which members of the groups 
+within A are distributed. 
+Demographic parity would then require that the prediction of the target  
+variable is statistically independent of sensitive attributes conditional 
+on D. Simply aggregating outcomes across high-level categories can be 
+misleading when the data can be further disaggregated. 
+More granular categories generally contain smaller sample sizes, and 
+it can be more difficult to establish that trends seen in very small 
+samples are not due to random chance. 
+It's important to review metrics across these more graular categories, 
+if they exist, to verify that disparate outcomes persist across all levels 
+of aggregation. 
+
+We also recommend watching out for the multiple comparisons problem, 
+which states that the more statistical inferences are made, the 
+more erroneous those inferences will become. For example, in the case 
+of evaluating fairness metrics on multiple groups, as we break the 
+groups down into more granular categories and evaluate those smaller 
+groups, it will become more likely that these subgroups will 
+differ enough to fail one of the metrics. For dealing with the multiple 
+comparisons problem, we recommend investigating statistical techniques 
+meant to correct the errors produced by individual statistical tests. 
```

Comment:
Can we add a link to a suggested technique? e.g., Bonferroni, Benjamini-Hochberg, etc

Reply:
Yes, linked to this site: https://www.statology.org/bonferroni-correction/